frontend/src/packages/survey/libs/constants/steps-default-order.constant.ts

```diff
@@ -0,0 +1,10 @@
+const STEPS_DEFAULT_ORDER = [
```

Comment:
we don't define separate file for each constant, move it to constants/constants

Reply:
done